com.unity.render-pipelines.universal/Runtime/NativeRenderPass.cs

```diff
@@ -692,6 +634,16 @@ internal static int GetValidPassIndexCount(int[] array)
             return array.Length - 1;
         }
 
+        internal static RTHandle GetFirstAllocedRTHandle(ScriptableRenderPass pass)
+        {
```

Comment:
nitpick: I would use Allocated instead of Alloced because it sounds weird :)

Reply:
Done in eb5919e